src/main/java/org/altbeacon/beacon/service/RangedBeacon.java

```diff
@@ -49,65 +56,26 @@ public void commitMeasurements() {
         }
     }
 
-	public static void setSampleExpirationMilliseconds(long milliseconds) {
```

Comment:
setSampleExpirationMilliseconds needs to be retained for backwards compatibility.  I suggest we change it to check to see if the filter is an instance of RunningAverageRssiFilter, and if so, call a method to set an equivalent value on the class.  If it is not, log a warning saying this will do nothing.